src/map/status.c

```diff
@@ -13339,10 +13339,12 @@ static void status_read_job_db(void)
 	int i = 0;
 	struct config_t job_db_conf;
 	struct config_setting_t *jdb = NULL;
+	char config_filename[256];
+	
 #ifdef RENEWAL_ASPD
-	const char *config_filename = "db/re/job_db.conf";
+	libconfig->format_db_path("re/job_db.conf", config_filename, sizeof(config_filename));
```

Comment:
probably here need use defines similar to ``DBPATH``?
For example ``DBPATHRE`` and ``DBPATHPRERE``